### PR TITLE
GGRC-1766 Add CustomRoles admin panel

### PR DESCRIPTION
**Pull request by plamut**
~~I do not know why I get an error _"instance.get_binding"_ is not a function, what am I missing?~~

~~I thought it was the missing `CMS.Models.CustomRoleDefinition` model (will add it soon), but even if I comment `cild_options` out of the descriptor, the error still occurs.~~

This PR adds the `AccessControlRole` on the frontend, and an admin panel for CRUD operations on it. Please check if the UI exposes all the fields end users should see, and nothing more (or less).

### CODE REVIEW

File: src/ggrc/assets/javascripts/models/mappers/mappings-ggrc.js [hunk 1]
```diff
@@ -496,6 +496,16 @@
         });
       }, 'CustomAttributeDefinition')
     },
+    // used by the Custom Roles admin panel tab
+    Roleable: {
+      custom_role_definitions: CustomFilter(
+        'WHAT_HERE???',  // TODO: or a different mapper helper? which?
```
Comment (plamut) [inline src/ggrc/assets/javascripts/models/mappers/mappings-ggrc.js]: @Smotko Blocked here, what "loader helper" (if that's a right term) should I use here? Did not find anything suitable, and not even `CustomFilter` worked, as it relies on another mapping to filter on it.

I basically just want to fetch `AccessControlRole` instances that belong to a chosen model type. Is there a good way to do it, or do I have to wait for the updates to Search API?

The model itself seems to work,  `AccessControlRole.findAll()` succeeds (with an empty list, though, b/c no data in my DB yet).

Comment (anze3db) [reply src/ggrc/assets/javascripts/models/mappers/mappings-ggrc.js]: Something like:

```javascript
Search(function (binding) {
  return CMS.Models.AccessControlRole.findAll();
}, 'AccessControlRole')
```

Should work here.

Comment (anze3db) [reply src/ggrc/assets/javascripts/models/mappers/mappings-ggrc.js]: You can also populate your database by doing 

```javascript
var acr = new CMS.Models.AccessControlRole({
  name: 'Test Control role',
  type: 'Control',
  read: true
});
acr.save();
```

This is also a good way to verify that your frontend model is set up correctly.

Or populate your database directly from mysql:

```mySQL
insert into access_control_roles (name, type) VALUES ('name', 'Control');
```

Comment (plamut) [reply src/ggrc/assets/javascripts/models/mappers/mappings-ggrc.js]: I've tried that initially, but got `can.Map: Object does not exist` error (when expanding a subtree). Or is that actually some different kind of error?

That is, when defining the `Search()` binding. BTW, didn't you say in an offline the search API does not work yet for `AccessControlRole` instances?

**Pull request merged**